app/acme.py

```diff
@@ -61,15 +61,15 @@ def account_request(self, request):
         protected = {
             "alg": "ES256",
             "nonce": self.nonce,
-            "url": self.url + "acme/new-acct",
+            "url": self.url + "acme/new-account",
```

Comment:
Would be nice to use the available `/directory` of the acme server.

Bit of json parsing.

See for example: https://acme.proeftuin.uzi-online.rdobeheer.nl/directory